### ISSUE TITLE
Arabic lines still display in English in Advanced Per-system Settings and Game Settings

### ISSUE THREAD

**Issue by Hew-ux** [batocera-linux/batocera-emulationstation#1634]
Despite the lines being translated, the various game settings remain in English:
![POEdit screenshot](https://user-images.githubusercontent.com/67527064/224519402-a62831ef-1ee6-404a-b960-3ed52902a5be.PNG)
![screenshot-2023 03 12-12h33 54](https://user-images.githubusercontent.com/67527064/224519405-52327c4d-6fba-4fc6-80f4-8c5189977b30.png)
This is true across the global Game Settings menu and the Advanced per-system options.

**Comment by fabricecaruso** [comment]
"Mar 12, 2023" ???? Did you check it's still happening ?
Translations are on batocera's side ( here https://github.com/batocera-linux/batocera.linux/blob/master/package/batocera/emulationstation/batocera-es-system/locales/ar/batocera-es-system.po ) and they look OK.

Can you confirm it's still happening with v39 beta ?

**Comment by n2qz** [comment]
@Hew-ux @Chipsum does the problem still exist?  Please test and confirm with latest v39 beta.

**Comment by sh4h33nDZ** [comment]
بسم الله
This problem (like the shown in the picture) is in the arabic support first, bcs all the letters in the os are sperated, and the po file is showing it correctly, so the source code must be edited to support letters overlay (تراكب الأحرف) and to use always the po file, not the system string
والله الموفق

**Comment by nadenislamarre** [comment]
in fact, it is just that some strings are not yet translated.
for the rest, all is translated on my side.

**Comment by sh4h33nDZ** [comment]
OK
But don't forget: character overlay
تراكب الأحرف

hope for you good results!!!

________________________________
من: Nicolas Adenis-Lamarre ***@***.***>
‏‏تم الإرسال: 16/رجب/1446 08:06 م
إلى: batocera-linux/batocera-emulationstation ***@***.***>
نسخة: sh4h33nDZ ***@***.***>; Comment ***@***.***>
‏‏الموضوع: Re: [batocera-linux/batocera-emulationstation] Arabic lines still display in English in Advanced Per-system Settings and Game Settings (Issue #1634)


in fact, it is just that some strings are not yet translated.
for the rest, all is translated on my side.

**Comment by Tartifless** [comment]
We received the request on discord to have cursive arabic in ES, it seems that cursive in the norm in arabic and that currently ES is in print script, here is an explanation i could find on a site explaining Arabic Alphabet:

_“Cursive” in the context of the Arabic alphabet:

So, what does it mean exactly when we say “cursive” in the context of the Arabic alphabet? In simple terms, cursive writing refers to a style of penmanship where the letters are written in a connected, flowing manner. This is different from print script, where letters are written separately. In Arabic, cursive is the norm, and each letter seamlessly connects to the next, creating a harmonious and visually appealing script._

If any developper is willing to help finalize the Arabic font with batocera-emulationstation, he would be more than welcome to help us, as none of the main developers are able to read Arabic.

**Comment by Kirito-87** [comment]
Anything new?

**Comment by nadenislamarre** [comment]
personnally i idon't understand if there is a problem and what is the problem.
it goes in all senses. for me it is normal that if there are untranslated strings, it mixes english and arabic.
2526 translations are missiing in the arabic batocera language.

**Comment by Tartifless** [comment]
The issue is not in the translation, nor in the fact that there is English.

The issue is that the Arabic characters are not displayed properly (maybe an issue with the font), they should be shown as cursive (like "attaché" in french) instead of separated characters.

**Comment by nadenislamarre** [comment]
i think different people talk about different subject : the tile is  "Arabic lines still display in English" but you says : "The issue is not in the translation, nor in the fact that there is English."

**Comment by Tartifless** [comment]
> i think different people talk about different subject : the tile is "Arabic lines still display in English" but you says : "The issue is not in the translation, nor in the fact that there is English."

Yes this is why I say it deserves a separate ticket @Kirito-87 @sh4h33nDZ

**Comment by sh4h33nDZ** [comment]
ok, i will open an issue for "attaché" arabic letters

**Comment by sh4h33nDZ** [comment]
Now, let's continue in problem of the Non-Translated strings
because i opened another issue for the arabic letters problem
